changelog/next/breaking-changes/3785.md

```diff
@@ -0,0 +1,2 @@
+The "autostart on stop" feature is not available in the pipeline manager
+anymore.
```

Comment:
I'd label this a bug fix rather than a breaking change, since it didn't make any sense to begin with.

Reply:
Changed in https://github.com/tenzir/tenzir/pull/3785/commits/7d736742695ef5e199b8577633277afa414161ca